Sources/StreamChat/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -270,6 +270,8 @@ extension ChatChannelListController: EventsControllerDelegate {
             linkChannelIfNeeded(messageNewEvent.channel)
         } else if let updatedChannelEvent = event as? ChannelUpdatedEvent {
             unlinkChannelIfNeeded(updatedChannelEvent.channel)
+        } else if let channelVisibleEvent = event as? ChannelVisibleEvent, let channel = dataStore.channel(cid: channelVisibleEvent.cid) {
+            linkChannelIfNeeded(channel)
```

Comment:
We should add a test case for this scenario, like we have for the other ones 👍

Reply:
done